tests/integration/test_interaction.py

```diff
@@ -581,3 +581,130 @@ def test_keyboard_help(self):
+        self.assertRegexpMatches(self._get_style(target_item_selector, 'left'), r'^\d+(\.\d+)?px$')
+        self.assertRegexpMatches(self._get_style(target_item_selector, 'top'), r'^\d+(\.\d+)?px$')
+
+    def test_zone_align_invalid(self):
+        """
+        Test items placed in a zone with an invalid align setting.
+        """
+        self._assert_zone_align_item(3, 'Zone Invalid Align', 'start')
+        self._assert_zone_align_item(4, 'Zone Invalid Align', 'start')
+        self._assert_zone_align_item(5, 'Zone Invalid Align', 'start')
+
+    def test_zone_align_invalid_key(self):
+        """
+        Test items placed in a zone with an invalid align setting, via keyboard.
+        """
+        self._assert_zone_align_item(3, 'Zone Invalid Align', 'start', Keys.RETURN)
+        self._assert_zone_align_item(4, 'Zone Invalid Align', 'start', Keys.RETURN)
+        self._assert_zone_align_item(5, 'Zone Invalid Align', 'start', Keys.RETURN)
+
+    def test_zone_align_left(self):
+        """
+        Test items placed in a zone with an left align setting.
+        """
+        self._assert_zone_align_item(6, 'Zone Left Align', 'left')
+        self._assert_zone_align_item(7, 'Zone Left Align', 'left')
+        self._assert_zone_align_item(8, 'Zone Left Align', 'left')
+
+    def test_zone_align_left_key(self):
+        """
+        Test items placed in a zone with an left align setting, via keyboard.
+        """
+        self._assert_zone_align_item(6, 'Zone Left Align', 'left', Keys.RETURN)
+        self._assert_zone_align_item(7, 'Zone Left Align', 'left', Keys.RETURN)
+        self._assert_zone_align_item(8, 'Zone Left Align', 'left', Keys.RETURN)
+
+    def test_zone_align_right(self):
+        """
+        Test items placed in a zone with a right align setting.
+        """
+        self._assert_zone_align_item(9, 'Zone Right Align', 'right')
+        self._assert_zone_align_item(10, 'Zone Right Align', 'right')
+        self._assert_zone_align_item(11, 'Zone Right Align', 'right')
+
+    def test_zone_align_right_key(self):
+        """
+        Test items placed in a zone with an right align setting, via keyboard.
+        """
+        self._assert_zone_align_item(9, 'Zone Right Align', 'right', Keys.RETURN)
+        self._assert_zone_align_item(10, 'Zone Right Align', 'right', Keys.RETURN)
+        self._assert_zone_align_item(11, 'Zone Right Align', 'right', Keys.RETURN)
+
+    def test_zone_align_center_key(self):
+        """
+        Test items placed in a zone with an center align setting, via keyboard.
+
+        note: Erratic bug in place_item(action_key=None) when there's too many zones?
+        Works consistently from keyboard and manual testing.
+        """
+        self._assert_zone_align_item(12, 'Zone Center Align', 'center', Keys.RETURN)
+        self._assert_zone_align_item(13, 'Zone Center Align', 'center', Keys.RETURN)
+        self._assert_zone_align_item(14, 'Zone Center Align', 'center', Keys.RETURN)
```

Comment:
@pomegranited In case you tried out my suggestion about using `ddt` to DRY out these tests and found that it didn't work, here is an updated version of the code:

```python
@ddt
class ZoneAlignInteractionTest(InteractionTestBase, BaseIntegrationTest):
    ACTION_KEYS = (None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
    # ...
    @data(
        ([3, 4, 5], "Zone Invalid Align", "start"),
        ([6, 7, 8], "Zone Left Align", "left"),
        ([9, 10, 11], "Zone Right Align", "right"),
        ([12, 13, 14], "Zone Center Align", "center"),
    )
    @unpack
    def test_zone_align(self, items, zone, alignment):
        reset = self._get_reset_button()
        for item in items:
            for action_key in self.ACTION_KEYS:
                self._assert_zone_align_item(item, zone, alignment, action_key)
                # Reset exercise
                self.scroll_down(pixels=200)
                reset.click()
                self.scroll_down(pixels=0)
```

I hadn't tested my original suggestion, but the code above works fine on my machine; no issues with `place_item`. You should be able to use it to replace the existing `test_zone_align_*` methods.